### PR TITLE
Add a Create Group command

### LINKED ISSUE #58: Add a Create Group command

### PR DESCRIPTION
**Pull request by voldiemort**
Fixes #58

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java [hunk 1]
```diff
@@ -157,6 +158,16 @@ public ObservableList<Person> getFilteredPersonList() {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public void addGroup(Group group) {
+            throw new AssertionError("This method should not be called.");
+        }
+
```
Comment (chongsun2002) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:167]: Im guessing these are just dummy methods that will be replaced in the near future.

**Review by chongsun2002 — APPROVED** [review]
LGTM!

**Review by seah-minlong — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java (hunk 1 above)
Comment (seah-minlong) [reply src/test/java/seedu/address/logic/commands/AddCommandTest.java:167]: I think this is the stub. All its methods throws the AssertionError 😆

File: src/main/java/seedu/address/logic/parser/group/CreateGroupCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * Parser for Commands of type CreateGroup.
+ */
+public class CreateGroupCommandParser implements Parser<CreateGroupCommand> {
+    @Override
+    public CreateGroupCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_MEMBERS);
+
+        String groupName = argMultimap.getPreamble();
+        String membersString = argMultimap.getValue(PREFIX_MEMBERS).orElse("");
```
[the remark this answers is not in the document]
Comment (voldiemort) [reply src/main/java/seedu/address/logic/parser/group/CreateGroupCommandParser.java:27]: Sounds good, will updated accordingly

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java (hunk 1 above)
Comment (voldiemort) [reply src/test/java/seedu/address/logic/commands/AddCommandTest.java:167]: This ModelStub is only for testing of the AddCommand which doesn't require this method. Future unit tests for this command will include more comprehensive testing

**Review by seah-minlong — APPROVED** [review]
LGTM! 💯

**Pull request merged**